releasenotes/notes/prepare-0.18.3-321aa7dc38f23e29.yaml

```
@@ -0,0 +1,5 @@
+---
+prelude: >
+  This bugfix release fixes a few minor issues in the 0.18 series, including a
+  performance regression in :obj:`~qiskit.compiler.assemble` when dealing with
+  non-pulse backends.
```

Just a small clarification it's a regression when we run a circuit on a pulse enabled ibm backend but aren't leveraging any of the pulse features:

```suggestion
  performance regression in :obj:`~qiskit.compiler.assemble` when dealing with
  executing :class:`~qiskit.circuit.QuantumCircuit` objects on pulse enabled
  backends.
```